frontend/src/pages/AccountReceive.vue

```diff
@@ -296,6 +322,8 @@ function useScan() {
         }
         // Wait for the first batch of web workers to finish scanning before creating new workers
         await firstScanPromise;
+        // Clear out existing workers
+        workers.length = 0;
```

Comment:
Just verifying.. we need this because we're gonna fall out and execute this asynce code upon workers being terminated by the 'Stop' button and the fact that workers array is instantiated now in this file and not in the workers.ts anymore means we need to empty the array here?